sdk/monitor/Azure.Monitor.OpenTelemetry.Exporter/src/Internals/AzMonNewListExtensions.cs

```diff
@@ -88,4 +147,17 @@ private static bool IsDefaultPort(int port)
     {
         return port == 0 || port == 80 || port == 443;
     }
+
+    [MethodImpl(MethodImplOptions.AggressiveInlining)]
+    private static bool TryGetNonDefaultPort(string? stringport, out string? port)
+    {
+        port = stringport;
+
+        if (string.IsNullOrEmpty(stringport) || stringport == "80" || stringport == "443")
+        {
+            return false;
+        }
+
+        return true;
+    }
```

Comment:
I can't find where this is used.

This doesn't look like a typical TryGet pattern.
Why does this need an out param?
If out is needed, should add the `NotNullWhenTrue` attribute.

Reply:
Fixed.